build/configs/rtl8730e/cpp_loadable_8M/defconfig

```diff
@@ -1434,7 +1434,9 @@ CONFIG_SYSTEM_PREAPP_STACKSIZE=2048
 CONFIG_SYSTEM_IPERF=y
```

Comment:
This commit doesn't follow the commit guidelines, please modify the title accordingly. And I think we can split this PR into two commits,

1. add missing initialization of adj_stack_ptr for armv7a
2. fix mem leak checker for rtl8730e and enable it by default